package.json

```diff
@@ -131,6 +131,8 @@
     "eject": "react-scripts eject",
     "lint:fix": "eslint --cache --fix --ext=js,jsx,ts,tsx src",
     "lint": "eslint --cache --ext=js,jsx,ts,tsx src",
+    "stylelint": "stylelint 'src/**/*.css'",
```

Comment:
I wonder if it's worth merging stylelint into the same script as lint, e.g. elslint && styelint